### PR TITLE
Storage: Adds `--all-projects` to `lxc storage volume list`

### LINKED ISSUE #10753: Implement all-projects query parameter for storage volumes
In order to complete https://github.com/canonical/lxd-cloud/issues/136, upstream servers need to synchronise instance and storage volume records when the make a connection to a downstream. 

To accomplish this, we can call `/1.0/instances?all-projects=true` for instances, but there is currently no equivalent for volumes. We could iterate over projects and make a separate call for each but this would result in lots of unnecessary requests.

### PR DESCRIPTION
**Pull request by tomponline**
Closes #10755
Closes #10753

### CODE REVIEW

File: lxc/storage_volume.go [hunk 1]
```diff
@@ -1340,8 +1341,9 @@ func (c *cmdStorageVolumeList) Command() *cobra.Command {
 	cmd.Aliases = []string{"ls"}
 	cmd.Short = i18n.G("List storage volumes")
 
-	c.defaultColumns = "tndcuL"
+	c.defaultColumns = "ptndcuL"
```
Comment (stgraber) [inline lxc/storage_volume.go:1344]: The new column should be mentioned in the help.
We may also want to use `e` to line it up with `lxc list`?

**Comment by stgraber** [comment]
@tomponline looks good, just need the tweak to the CLI to have it covered in help and potentially line it up with `lxc list` if that makes sense.

File: lxc/storage_volume.go (hunk 1 above)
Comment (tomponline) [reply lxc/storage_volume.go:1344]: @stgraber I've remove the "Default column layout is: tndcuL" bit (as this wasn't present in `lxc list` and is already provided in the default value for the `-c` flag option. This saves having to maintain it in 2 places.

I've rearranged the documented column names into alphabetical order (like `lxc list` has them).
However `lxc list` is missing a letter column for "Project", and both "p" and "P" are taken for other things.
So we need to pick a letter for Project on "lxc list" and then use that with "lxc storage volume list" as well.

Comment (tomponline) [reply lxc/storage_volume.go:1344]: Oh no, sorry I missed it, its "e" for "Project name". OK will use that here too.

**Pull request merged**